src/FighterStats.cpp

```diff
@@ -3,42 +3,59 @@
 
 using std::to_string;
 
-//FIXME: Trocar side pra enum
-FighterStats::FighterStats(Fighter *p_fighter, int p_index_fighter, int p_side, double p_x, double p_y){
+/** @file arquivo.h */
+
+/**
+ * Verify the fighter side and update the background acording
+ * to the life and ultimate stats.
+ */
+FighterStats::FighterStats(Fighter *p_fighter, int p_index_fighter, 
+	int p_side, double p_x, double p_y){
 	fighter = p_fighter;
 	side = p_side;
-	x = p_x;
+	x = p_x; 
 	y = p_y;
 	percent_to_draw_life = 1.0;
 	index_fighter = p_index_fighter;
 
 	is_ultimate_diff = fighter->is("flesh");
 
+	/**
+	 * Create the fighter life bar and ultimate on the background.
+	*/
 	bg[0] = Sprite("hud/life" + to_string(index_fighter) + "_frame.png");
-	bg[1] = bg[2] = Sprite("hud/life" + to_string(index_fighter) + "_frame.png");
-	if(is_ultimate_diff) bg[2] = Sprite("hud/life" + to_string(index_fighter) + "_frame_ultimate.png");
-
-	//Left
-	if(side == 0){
+	bg[1] = bg[2] = Sprite("hud/life" + to_string(index_fighter) + 
+		"_frame.png");
+	if (is_ultimate_diff) bg[2] = Sprite("hud/life" + to_string(index_fighter) 
+		+ "_frame_ultimate.png");
+
+	/**
+	 * Create the left side of the background life and ultimate.
+	*/
+	if (side == 0) {
 		empty_bg[0] = Sprite("hud/empty_background.png");
 		empty_bg[1] = empty_bg[2] = Sprite("hud/empty_background.png");
-		if(is_ultimate_diff) empty_bg[2] = Sprite("hud/empty_background_ultimate.png");
+		if (is_ultimate_diff) empty_bg[2] = 
+			Sprite("hud/empty_background_ultimate.png");
 		life[0] = life[2] = Sprite("hud/left_life.png");
 		life[1] = Sprite("hud/full_left_life.png");
-		if(is_ultimate_diff) life[2] = Sprite("hud/in_ultimate_left_life.png");
+		if (is_ultimate_diff) life[2] = Sprite("hud/in_ultimate_left_life.png");
 		special[0] = Sprite("hud/left_special_bar.png");
 		special[1] = Sprite("hud/full_left_special_bar.png");
 		special[2] = Sprite("hud/in_ultimate_left_special_bar.png");
 	}
 
-	//Right
-	if(side == 1){
+	/**
+	 * Create the right side of the background life and ultimate.
+	*/
+	if (side == 1) {
 		empty_bg[0] = Sprite("hud/empty_background.png");
 		empty_bg[1] = empty_bg[2] = Sprite("hud/empty_background.png");
-		if(is_ultimate_diff) empty_bg[2] = Sprite("hud/empty_background_ultimate.png");
+		if (is_ultimate_diff) empty_bg[2] = 
+			Sprite("hud/empty_background_ultimate.png");
```

Comment:
Put all the attribution on line below, like: `empty_bg[2] = Sprite("hud/empty_background_ultimate.png");` (leave just the if on the line above)